lib/tasks/fractal-build.js

```diff
@@ -16,11 +17,26 @@ const logger = fractal.cli.console
  * configuration option set above.
  */
 
-gulp.task('fractal:build', function () {
+gulp.task('fractal:build', ['fractal:assets'], function () {
   const builder = fractal.web.builder()
   builder.on('progress', (completed, total) => logger.update(`Exported ${completed} of ${total} items`, 'info'))
   builder.on('error', err => logger.error(err.message))
+
   return builder.build().then(() => {
     logger.success('Fractal build completed!')
+
+    // Fix theme assets not being copied, see below
+    copyThemeAssets()
   })
 })
+
+/*
+ * Copy the theme assets to the static build location
+ *
+ * Fractal should do this, but it looks like a bug. Mentioned to
```

Comment:
No - I'll open one tomorrow. I'd just discussed it with Mark.

In the mean it's handy to have a working static build tho'